### PR TITLE
The -o option can be used multiple times

### PR DESCRIPTION
**Pull request by milo**
For example:

```
vendor/bin/tester -o console -o tap:output.tap -o junit:output.xml -o log:output.log
```

Tester ASCII art logo is shown, only if output for stdout (without parameter) is not passed.

### CODE REVIEW

**Comment by dg** [comment]
Would not it be easier to leave CommandLine as is, change parameter to repeatable and split it in loadOptions()?

**Comment by milo** [comment]
@dg Probably yes. The `-o value:file/path` syntax seems ok? So, drop `CommandLine::ENUM` (this is the only one place) and parse it manually. Or add `CommandLine::NORMALIZER` like:
```php
CommandLine::NORMALIZER => function ($arg) {
    list($value, $path) = explode(':', $arg, 2);
    if (...) throw ...
};
```
I'll try it, how it will look like.

**Comment by milo** [comment]
@dg What about such approach (last commit)?

**Comment by dg** [comment]
I tried to find if there is any pattern how should these parameters look, but it seems that this is pole neorané … not standardized. Sometimes it is used this way:

```
vendor/bin/tester -o console -o tap -file output.tap -o junit -file output.xml -o log -file output.log
```

Which looks little bit weird. So syntax looks fine to me.

File: src/Runner/CliTester.php [hunk 1]
```diff
@@ -96,21 +99,22 @@ private function loadOptions()
...
-    --coverage-src <path>        Path to source code.
-    -h | --help                  This help.
+    -p <path>                Specify PHP interpreter to run (default: php).
+    -c <path>                Look for php.ini file (or look in directory) <path>.
+    -d <key=value>...        Define INI entry 'key' with value 'val'.
+    -s                       Show information about skipped tests.
+    --stop-on-fail           Stop execution upon the first failure.
+    -j <num>                 Run <num> jobs in parallel (default: 8).
+    -o <console|tap|junit|log|none>
+                             Specify output format. Repeatable with parameter.
```
Comment (dg) [inline src/Runner/CliTester.php]: `Specify one or more output formats with optional file name` is probably better that `Repeatable with parameter`.

**Comment by dg** [comment]
In fact ENUM is used by Tester, try `tester --colors x` before and after.

**Comment by milo** [comment]
> In fact ENUM is used by Tester, try tester --colors x before and after.

Ups, I was rash. I have to find a way how to preserve `console|tap|junit|log|none` at one place and do not collide with ENUM implementation. Or replace ENUM implementation by something similarly easy.

**Comment by milo** [comment]
Closing in favour of #329

**Comment by milo** [comment]
Closing in favour of #329

**Pull request closed without merging**